src/test/java/api/RelatedInstanceTest.java

```diff
@@ -0,0 +1,308 @@
+package api;
+
+import static api.support.InstanceSamples.nod;
+import static io.vertx.core.json.JsonObject.mapFrom;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import org.folio.inventory.domain.instances.Instance;
+import org.folio.inventory.domain.instances.RelatedInstance;
+import org.junit.Test;
+
+import api.support.ApiTests;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+public class RelatedInstanceTest extends ApiTests {
+
+  @Test
+  public void testRelatedInstance() {
```

Comment:
What does this test about a related instance?

Reply:
Tests the static method that swap instanceId and relatedInstanceId appropriately.